src/bin/sage-env

```diff
@@ -251,6 +251,21 @@ if [ -z "${SAGE_ORIG_PATH_SET}" ]; then
     SAGE_ORIG_PATH_SET=True && export SAGE_ORIG_PATH_SET
 fi
 if [ -n "$SAGE_LOCAL" ]; then
+    # https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html
+    MAMBA_ROOT_PREFIX="$SAGE_LOCAL/var/lib/sage/conda"
+    CONDA_DEFAULT_ENV=sage
+    CONDA_PREFIX="$MAMBA_ROOT_PREFIX/envs/$CONDA_DEFAULT_ENV"
```

Comment:
Do we need two different prefixes? `CONDA_PREFIX` can just be equal to `MAMBA_ROOT_PREFIX`

Reply:
Thanks, I'll try this. Didn't find good documentation on the environment variables that take effect

Reply:
Done in 7c67f69